src/components/common/CertifySection.vue

```diff
@@ -52,6 +55,9 @@ export default class CertifySection extends Mixins(DateMixin, SharedMixin) {
   /** Whether to perform validation. */
   @Prop({ default: false }) readonly validate: boolean
 
+  /** To determine whether user input is enabled. */
+  @Prop({ default: false }) readonly disableEdit: boolean
```

Comment:
This default will be used when this component is used without specifying this prop.

What is the requirement for the Alteration page and the Correction page? To enable edits or disable them?

Reply:
I think you're right here: it needs to be enabled (not disabled) for those other pages because we don't pre-populate the user's name.

Resolving.